### ISSUE TITLE
Changing `brand-color` doesn’t change the yellow background?

### ISSUE THREAD

**Issue by 0xtetus** [patdryburgh/hitchens#28]
I tried changing the color value in `/_sass/_variables.scss` but the background color remains unchanged. What am I doing wrong?

**Comment by sheephunt2000** [comment]
It seems like you resolved this, but for everyone else checking in, I had the same issue. Just make another commit to the site -- this will make Github rebuild the site and it should work perfectly.

**Comment by ellepannitto** [comment]
I'm trying to use this template for a research project. We edited the brand-color multiple times (we switched from yellow to green) but the compiled css is not changing: https://github.com/LaboratorioSperimentale/DiCCAS-interface

What bugs me the most is that also the new page we added (`corpus.md`) is not being compiled to html. The new item appears in the menu but points to nothing, and I can't see the website data in the repo anywhere. I don't have a lot of experience with jekyll but I did use it for other projects and there was always a `site` folder or something like that with the compiled files. What am I doing wrong?

Thank you!
